examples/renameimage.py

```diff
@@ -38,44 +97,100 @@
 
     glancesync = GlanceSync()
     glancesync.init_logs()
+    master = glancesync.master_region
+
+    master_images = glancesync.get_images_region(master)
+
+    old_image = None
+    new_image = None
+    for image in master_images:
+        if image.name == sys.argv[1]:
+            if old_image:
+                msg = 'There are more than an image with the old name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                old_image = image
+        elif image.name == sys.argv[2]:
+            if new_image:
+                msg = 'There are more than an image with the new name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
```

Comment:
Because it is a script not a module. Scripts does not raise exceptions, print a message and exit with an error code. Exceptions are for developers not for users, in general a program that shows an exception (not in debug mode) instead of a clean error message and an error code, IMHO make a poor impression, of a not finish work.